data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -115,30 +118,48 @@
       "note": [
         ""
       ],
-      "attested_entities": [
-        {
-          "entity_type": "CV",
-          "role": "CV",
-          "id": "URI/ARK",
-          "attested_name_NS": "",
-          "attested_name": "",
-          "note": [
-            "array of strings"
-          ]
-        }
-      ]
-    }
-  ],
-  "associated_entities": [
-    {
-      "entity_type": "CV",
-      "relation": "CV",
-      "id": "URI/ARK",
-      "note": [
-        "array of strings"
-      ]
+      "attested_entities": {
+        "attested_person": [
+          {
+            "relation": "CV",
+            "id": "URI",
+            "note": [
+              "array of strings"
+            ]
+          }
+        ],
+        "attested_place": [
+          {
+            "relation": "CV",
+            "id": "URI",
+            "note": [
+              "array of strings"
+            ]
+          }
+        ]
+      }
     }
   ],
+  "associated_entities": {
+    "associated_person": [
+      {
+        "relation": "CV",
+        "id": "URI",
+        "note": [
+          "array of strings"
+        ]
+      }
+    ],
+    "associated_place": [
+      {
+        "relation": "CV",
+        "id": "URI",
+        "note": [
+          "array of strings"
+        ]
+      }
+    ]
+  },
   "associated_dates": [
```

Comment:
I wonder if for now we need to decide between 'event' and 'associated_date'? Based on our current data and what our phase 1 scope is, I think 'associated_date' is the better field. We could add additional fields to it for something like `date_ns` (transcribing the date from the ms), some indication of the dating system used by the date as found in the manuscript, etc. For now we could include source info (such as, 'Found in scribal colophon') in the note field.

Rather than using the type field, we could structure this field like the related objects:

```
{
  ...
  "associated_dates":
    {
      "creation": [array of objects further specifying, if needed, a date],
      "acquisition": [object describing the acquisition.]
    }
}
```

I think using a dates field will be easier for initial data gathering, as its focus is on the information _on the page_ (the date transcribed from the ms). However, it would be difficult to start associating together people, places, and dates all occurring in the same paratext that is describing/providing evidence for an event.